P/PHYLIP/build_tarballs.jl

```diff
@@ -0,0 +1,92 @@
+# Note that this script can accept some limited command-line arguments, run
+# `julia build_tarballs.jl --help` to see a usage message.
+using BinaryBuilder, Pkg
+
+name = "PHYLIP"
+version = v"3.697.0"
+
+# Collection of sources required to complete build
+sources = [
+    ArchiveSource("http://evolution.gs.washington.edu/phylip/download/phylip-$(version.major).$(version.minor).tar.gz", "9a26d8b08b8afea7f708509ef41df484003101eaf4beceb5cf7851eb940510c1")
+]
+
+# Bash recipe for building across all platforms
+script = raw"""
+cd $WORKSPACE/srcdir
+cd phylip-*/src/
+make -f Makefile.unx install
+mkdir -p $bindir $libdir
+mv ../exe/libdrawtree.so $libdir/
+mv ../exe/libdrawgram.so $libdir/
```

Comment:
Are these actually called `.so` on other platforms (like macOS)?  If not, you can use `${dlext}` as a convenient bash variable to get `.so`, `.dylib` and `.dll` as appropriate.

Reply:
No, better to look at the actual error I'd say. Eg for macos:
`[18:26:03] bash: line 1: /Developer/Tools/Rez: No such file or directory`